### ISSUE TITLE
investigate remaining servlet overlays

### ISSUE THREAD

**Issue by pnicolucci** [OpenLiberty/open-liberty#14799]
This is a follow on to: https://github.com/OpenLiberty/open-liberty/issues/14796 /  #12637 to investigate:

```
	datatypes.dtd
	xml.xsd
	XMLSchema.dtd
```

**Comment by volosied** [comment]
I think there files are used for web-app parsing or validation?  

The extension means they're fairly old. From what I read, XSD "is preferred since JSP 2.0 / Servlets 2.4". 

https://www.oracle.com/webfolder/technetwork/jsc/xml/ns/javaee/index.html#3 shows some information on the dtd files. 

Also, Java EE 5 Schema Resources use XSD as well. 

datatypes.dtd is used by XMLSchema.dtd:
https://github.com/OpenLiberty/open-liberty/blob/464735a8588770d8ce055a4505f53c9a029b12ab/dev/com.ibm.ws.webcontainer/data/cache/www.w3.org/2001/XMLSchema.dtd#L18

xml.xsd seems to reference XMLSchema.dtd:
https://github.com/OpenLiberty/open-liberty/blob/464735a8588770d8ce055a4505f53c9a029b12ab/dev/com.ibm.ws.webcontainer/data/cache/www.w3.org/2001/xml.xsd#L2

I'm assuming these dtd files are here for old servlet/namespace support?  
I discovered that there's both
- https://www.w3.org/2001/XMLSchema.dtd
- https://www.w3.org/2001/XMLSchema.xsd

**Comment by volosied** [comment]
Tomcat also includes these files. I think we should also let them be: 
https://github.com/apache/tomcat/tree/master/java/jakarta/servlet/resources

**Comment by pnicolucci** [comment]
In addition our JSP implementation lists the following in this class: [TldParser](https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/TldParser.java):

```
    public static final String XMLSCHEMA_DTD_PUBLIC_ID = "-//W3C//DTD XMLSCHEMA 200102//EN";
    public static final String XMLSCHEMA_DTD_RESOURCE_PATH = "/javax/servlet/resources/XMLSchema.dtd";
    
    public static final String DATATYPES_DTD_PUBLIC_ID = "datatypes";
    public static final String DATATYPES_DTD_RESOURCE_PATH = "/javax/servlet/resources/datatypes.dtd";
    
    public static final String XML_XSD_SYSTEM_ID = "http://www.w3.org/2001/xml.xsd";
    public static final String XML_XSD_RESOURCE_PATH = "/javax/servlet/resources/xml.xsd";
```

**Comment by pnicolucci** [comment]
We could potentially open an issue with the Servlet group to see about adding these in a future release so we don't have to maintain them in our version of the API JAR

**Comment by pnicolucci** [comment]
Files included for servlet 5 are the same files included in servlet 4.0 and previous.
